src/olympia/abuse/migrations/0019_alter_abusereport_drop_state_options_alter_abusereport_managers.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.8 on 2023-12-12 17:59
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('abuse', '0018_cinderpolicy_cinderjob_policies'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='abusereport',
+            options={},
+        ),
+        migrations.AlterModelManagers(
+            name='abusereport',
+            managers=[
+            ],
+        ),
+        migrations.RemoveField(
+            model_name='abusereport',
+            name='state',
+        ),
```

Comment:
Field was still there in the model before this commit so if you drop the database column now all `create()` and `save()` on `AbuseReport` will fail during the push.

Reply:
forgot about that - https://github.com/mozilla/addons-server/pull/21611/commits/69fb65a5cc1a66f04a80daafcc4f7c0ec0398ab7